lib/poller.js

```diff
@@ -49,6 +50,32 @@ class Poller extends EventEmitter {
       this.interval = null
     }
   }
+
+  getCachedSerializedDataScript () {
```

Comment:
One potential way to clean this up:
rename this function generateSerializedData and have it return just the JSON.stringified data

move the <script> parts back to data-serializer